automated-demo/auto-demo.sh

```diff
@@ -666,43 +671,53 @@ mongo_pacman_demo()
   echo "10. We need a route for our Pacman application, let's create a FederatedIngress that points to our LoadBalancer"
   wait_for_input
   run_ok_or_fail "oc --context=feddemocl1 -n ${DEMO_NAMESPACE} create -f yaml-resources/pacman/03-pacman-federated-ingress-mod.yaml" "0" "1"
-  echo "11. Now we are ready to deploy the Pacman application accross the clusters"
+  echo "11. We need a route for our Pacman application, let's create a FederatedIngress that points to our LoadBalancer"
```

Comment:
Change logging, it seems you're creating a FederatedServiceaccount and not a FederatedIngress.